commit 3cee6b56c6df129cd70cec9748f98738e845ee75
Author: Nicolas Lécureuil <neoclust@mageia.org>
Date:   Mon Sep 25 15:53:21 2017 +0200

    Disable doc folder for the moment

diff --git a/Makefile.am b/Makefile.am
index 430e3dd59..61cd46856 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -23,7 +23,7 @@
 ACLOCAL_AMFLAGS = -I m4
 DISTCHECK_CONFIGURE_FLAGS=--disable-conf-backup --disable-python-check
 
-SUBDIRS = web services doc
+SUBDIRS = web services
 
 backupext = .$(shell date +%Y-%m-%d+%H:%M:%S)
 
