diff --git a/web/modules/report/report/get_file.php b/web/modules/report/report/get_file.php
index 8500349..be9aa77 100644
--- a/web/modules/report/report/get_file.php
+++ b/web/modules/report/report/get_file.php
@@ -53,10 +53,6 @@ function readfile_chunked($filename, $retbytes = true) {
     return $status;
 }
 
-function startswith($haystack, $needle) {
-    return substr($haystack, 0, strlen($needle)) === $needle;
-}
-
 if (isset($_GET['path']) && startswith($_GET['path'], '/var/tmp/report-')) {
     // Prevent download to stop after PHP timeout
     set_time_limit(0);
