diff -Naur mmc-agent-2.3.1/conf/plugins/base.ini mmc-agent-2.3.1.new/conf/plugins/base.ini
--- mmc-agent-2.3.1/conf/plugins/base.ini	2008-05-19 11:40:42.000000000 +0200
+++ mmc-agent-2.3.1.new/conf/plugins/base.ini	2008-05-19 11:57:21.000000000 +0200
@@ -24,7 +24,7 @@
 # group gid number start
 gidStart = 10000
 # LDAP log file path
-logfile = /var/log/ldap.log
+logfile = /var/log/ldap/ldap.log
 # FDS log file path
 # logfile = /opt/fedora-ds/slapd-hostname/logs/access 
 
diff -Naur mmc-agent-2.3.1/conf/plugins/network.ini mmc-agent-2.3.1.new/conf/plugins/network.ini
--- mmc-agent-2.3.1/conf/plugins/network.ini	2008-05-19 11:40:42.000000000 +0200
+++ mmc-agent-2.3.1.new/conf/plugins/network.ini	2008-05-19 11:57:21.000000000 +0200
@@ -3,17 +3,17 @@
 
 [dhcp]
 dn = ou=DHCP,dc=mandriva,dc=com
-pidfile = /var/run/dhcpd.pid
-init = /etc/init.d/dhcp3-server
-logfile = /var/log/syslog
-leases = /var/lib/dhcp3/dhcpd.leases
+pidfile = /var/run/dhcpd/dhcpd.pid
+init = /etc/rc.d/init.d/dhcpd
+logfile = /var/log/messages
+leases = /var/lib/dhcp/dhcpd.leases
 
 [dns]
 dn = ou=DNS,dc=mandriva,dc=com
-pidfile = /var/run/bind/run/named.pid
-init = /etc/init.d/bind9
-logfile = /var/log/syslog
-bindroot = /etc/bind/
-bindgroup = bind
+pidfile = /var/lib/named/var/run/named.pid
+init = /etc/rc.d/init.d/named
+logfile = /var/log/messages
+bindroot = /var/lib/named
+bindgroup = named
 # dnsreader = DNS Reader
 # dnsreaderpassword = DNSReaderPassword
diff -Naur mmc-agent-2.3.1/conf/plugins/proxy.ini mmc-agent-2.3.1.new/conf/plugins/proxy.ini
--- mmc-agent-2.3.1/conf/plugins/proxy.ini	2008-05-19 11:40:42.000000000 +0200
+++ mmc-agent-2.3.1.new/conf/plugins/proxy.ini	2008-05-19 11:57:21.000000000 +0200
@@ -3,7 +3,7 @@
 
 [squidguard]
 path = /usr/bin/squidGuard
-blacklist = /var/lib/squidguard/db/bad.destdomainlist
-scriptReload = /etc/init.d/squid reload
-user = proxy
-group = proxy
+blacklist = /usr/share/squidGuard-1.2.0/db/bad.destdomainlist
+scriptReload = /etc/rc.d/init.d/squid reload
+user = squid
+group = squid
diff -Naur mmc-agent-2.3.1/conf/plugins/samba.ini mmc-agent-2.3.1.new/conf/plugins/samba.ini
--- mmc-agent-2.3.1/conf/plugins/samba.ini	2008-05-19 11:40:42.000000000 +0200
+++ mmc-agent-2.3.1.new/conf/plugins/samba.ini	2008-05-19 11:57:21.000000000 +0200
@@ -3,7 +3,7 @@
 # Computers Locations
 baseComputersDN = ou=Computers, dc=mandriva, dc=com
 sambaConfFile = /etc/samba/smb.conf
-sambaInitScript = /etc/init.d/samba
+sambaInitScript = /etc/rc.d/init.d/smb
 sambaClamavSo = /usr/lib/samba/vfs/vscan-clamav.so
 # Default SAMBA shares location
 defaultSharesPath = /home/samba
