--- Makefile	2007-09-10 17:52:26.000000000 +0200
+++ Makefile.oden	2007-09-10 18:34:26.000000000 +0200
@@ -20,11 +20,11 @@
 
 # General Makefile variables
 DESTDIR = 
-PREFIX = /usr/local
+PREFIX = /usr
 SBINDIR = $(PREFIX)/sbin
 LIBDIR = $(PREFIX)/lib/mmc
 ETCDIR = /etc/mmc
-INITDIR = /etc/init.d
+INITDIR = /etc/rc.d/init.d
 INSTALL = $(shell which install)
 SED = $(shell which sed)
 
@@ -57,11 +57,11 @@
 	-[ -d $(DESTDIR)$(ETCDIR) ] && mv -f $(DESTDIR)$(ETCDIR) $(DESTDIR)$(ETCDIR)$(BACKUP)
 	@echo ""
 	@echo "Creating directories..."
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(SBINDIR)
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(LIBDIR)
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(LIBDIRBACKUP)
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(ETCDIR)
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(PYTHON_PREFIX)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(SBINDIR)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(LIBDIR)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(LIBDIRBACKUP)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(ETCDIR)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(PYTHON_PREFIX)
 
 	@echo ""
 	@echo "Install python code in $(DESTDIR)$(PYTHON_PREFIX)"
@@ -69,27 +69,27 @@
 
 	@echo ""
 	@echo "Install LIBFILES in $(DESTDIR)$(LIBDIR)"
-	$(INSTALL) $(LIBFILES) -m 755 -o root -g root $(DESTDIR)$(LIBDIR)
+	$(INSTALL) $(LIBFILES) -m 755 $(DESTDIR)$(LIBDIR)
 
 	@echo ""
 	@echo "Install LIBDIRBACKUP in $(DESTDIR)$(LIBDIRBACKUP)"
 
-	$(INSTALL) $(LIBFILESBACKUP) -m 755 -o root -g root $(DESTDIR)$(LIBDIRBACKUP)
+	$(INSTALL) $(LIBFILESBACKUP) -m 755 $(DESTDIR)$(LIBDIRBACKUP)
 
 	@echo ""
 	@echo "Install SBINFILES in $(DESTDIR)$(SBINDIR)"
-	$(INSTALL) $(SBINFILES) -m 755 -o root -g root $(DESTDIR)$(SBINDIR)
+	$(INSTALL) $(SBINFILES) -m 755 $(DESTDIR)$(SBINDIR)
 
 	@echo ""
 	@echo "Install CONFILES in $(DESTDIR)$(ETCDIR)"
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(ETCDIR)/agent
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(ETCDIR)/agent/keys
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(ETCDIR)/plugins
-	$(INSTALL) conf/agent/config.ini -m 600 -o root -g root $(DESTDIR)$(ETCDIR)/agent
-	$(INSTALL) conf/agent/keys/* -m 600 -o root -g root $(DESTDIR)$(ETCDIR)/agent/keys
-	$(INSTALL) conf/plugins/* -m 600 -o root -g root $(DESTDIR)$(ETCDIR)/plugins
-	$(INSTALL) -d -m 755 -o root -g root $(DESTDIR)$(INITDIR)
-	$(INSTALL) -m 755 -o root -g root init.d/mmc-agent $(DESTDIR)$(INITDIR)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(ETCDIR)/agent
+	$(INSTALL) -d -m 755 $(DESTDIR)$(ETCDIR)/agent/keys
+	$(INSTALL) -d -m 755 $(DESTDIR)$(ETCDIR)/plugins
+	$(INSTALL) conf/agent/config.ini -m 600 $(DESTDIR)$(ETCDIR)/agent
+	$(INSTALL) conf/agent/keys/* -m 600 $(DESTDIR)$(ETCDIR)/agent/keys
+	$(INSTALL) conf/plugins/* -m 600 $(DESTDIR)$(ETCDIR)/plugins
+	$(INSTALL) -d -m 755 $(DESTDIR)$(INITDIR)
+	$(INSTALL) -m 755 init.d/mmc-agent $(DESTDIR)$(INITDIR)
 	$(SED) -i 's!^path[ \t].*$$!path = $(LIBDIR)/backup-tools!' $(DESTDIR)$(ETCDIR)/plugins/base.ini
 	$(SED) -i 's!##SBINDIR##!$(SBINDIR)!' $(DESTDIR)$(INITDIR)/mmc-agent
 
